Feature/cleanup docs (#461)
jjm3x3/AmazingRaceFantasy: README.md

1. Run either `seed:dev` or `seed:prod` depending which you need. 
    1. These have different credentials. Make sure you have the appropriate ones.
    1. Make sure you seed prod before you deploy updated code to prod.

## Using The Site

### Nromal Usage

> A any user can navigate to our various read-only pages.

Reviewer: I think you mean "Any user", but I'm curious if you mean any as a role here. If that's the case, I'd reword to something like "a user without a role" or something

Author: Yup I think cleaning it up to be just "Any user" makes sense

```suggestion
Any user can navigate to our various read-only pages.
```

### Create A League

If you are a user that would like to create a league will you will first need to have an account.

#### How To

1. Create an account, by going to our [Create Account Page](https://xfactorleague.com/create-account)

> 1. Let @jjm3x3 know, and he can get you setup

Reviewer: ```suggestion
1. Let @jjm3x3 know and he can get you set up.
```

> 1. Navigate to the [Create Creation Page](https://xfactorleague.com/league/configuration), fill out the form, and hit submit

Author: ```suggestion
1. Navigate to the [League Creation Page](https://xfactorleague.com/league/configuration), fill out the form, and hit submit
```

> 1. Navigate to the [Create Creation Page](https://xfactorleague.com/league/configuration) fill out the form and hit submit

Reviewer: ```suggestion
1. Navigate to the [Create Creation Page](https://xfactorleague.com/league/configuration), fill out the form, and hit submit
```
